Makefile

```diff
@@ -65,16 +65,25 @@ cephcsi:
 	if [ ! -d ./vendor ]; then (go mod tidy && go mod vendor); fi
 	GOOS=linux GOARCH=$(GOARCH) go build -mod vendor -a -ldflags '$(LDFLAGS)' -o _output/cephcsi ./cmd/
 
-.PHONY: containerized-build
+.PHONY: containerized-build containerized-check
```

Comment:
containerized-check where this is defined?

Reply:
Ah, should be `containerized-test`

Reply:
done